brother_ql/backends/helpers.py

```diff
@@ -131,21 +130,142 @@ def status(
     be = backend_factory(selected_backend)
     BrotherQLBackend = be["backend_class"]
     printer = BrotherQLBackend(printer_identifier)
+    return printer
 
-    logger.info("Sending status information request to the printer.")
-    printer.write(b"\x1b\x69\x53")  # "ESC i S" Status information request
+def get_status(
```

Comment:
C (non blocking): I will re-add a `status` function with a depreciation notice to ease the migration.

Reply:
I feel like this is not necessary since it was introduced less than 48 hours ago.

Reply:
Oh, overlooked that. Good catch - will remove the warning